src/current/v22.2/start-a-local-cluster.md

```diff
@@ -16,7 +16,11 @@ Once you've [installed CockroachDB](install-cockroachdb.html), it's simple to ru
 
 - Make sure you have already [installed CockroachDB](install-cockroachdb.html).
 - For quick SQL testing or app development, consider [running a single-node cluster](cockroach-start-single-node.html) instead.
-- Note that running multiple nodes on a single host is useful for testing CockroachDB, but it's not suitable for production. To run a physically distributed cluster, see [Manual Deployment](manual-deployment.html) or [Orchestrated Deployment](kubernetes-overview.html), and review the [Production Checklist](recommended-production-settings.html).
+- Running multiple nodes on a single host is useful for testing CockroachDB, but it's not suitable for production. To run a physically distributed cluster, refer to [Manual Deployment](manual-deployment.html) or [Orchestrated Deployment](kubernetes-overview.html), and review the [Production Checklist](recommended-production-settings.html).
+
+{{site.data.alerts.callout_danger}}
+A node's store directory contains cluster data, metadata, and logs. If the store is incompatible with the CockroachDB binary version, this can lead to panics or other problems when starting a node. If you previously started a cluster with a different major version of CockroachDB on a node, do not attempt to reuse the store location when starting a new cluster with a new CockroachDB version. Instead, either [upgrade the existing cluster]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}) to the new version to update the store directory's contents, or move or delete the previous store directory before starting the `cockroach` process. The store directory is the `cockroach-data/` directory in the same directory as the `cockroach` command by default, or the location passed to the `--store` flag otherwise. For details about the store location, refer to [cockroach start]({% link {{ page.version.version }}/cockroach-start.md %}#store).
```

Comment:
I would rephrase the first two sentences as follows,

> If the previously initialized store is incompatible with either the new CockroachDB binary or the new cluster configuration (e.g., `--start-single-node` disables replication), this can lead to panics or other problems when starting a cluster. Instead, either move or delete the previous store directory before starting the `cockroach` process.

The stuff about `upgrade` is a bit more subtle. An incompatible binary would exit with an error message, i.e., not panic; otherwise, an auto upgrade kicks in. So, it's best to leave it underspecified. The warning is mainly about incompatible cluster configurations between runs.

Reply:
Thanks, I took your suggestion and reworked it a bit.